src/config.ts

```diff
@@ -26,31 +26,40 @@ const config = {
     //Mozilla Auth Proxy supports a larger number of user groups for a user.
     cognito: {
       jwtIssuer:
+        // COGNITO_JWT_ISSUER is not set in this repo (or anywhere?)
         process.env.COGNITO_JWT_ISSUER ||
         'cognito-idp.us-east-1.amazonaws.com/us-east-1_1alKls4qw',
+      // COGNITO_KIDS is not set in this repo (or anywhere?)
```

Comment:
Maybe the env is meant to override it for local development or testing? (Just a guess.)

Reply:
best guess here is premature optimization. there were probably talks of creating env vars but it never happened.